### ISSUE TITLE
Strict NAT with non-EE SIM

### ISSUE THREAD

**Issue by mattswatson** [jamesmacwhite/hh70-ee#3]
I originally got this router as I had a 4G plan with EE. However, I've since moved to Three and am trying to use their SIM in the (now unlocked) device.

I believe, when using the 3internet APN, Three don't use a CGNAT. However, I'm still getting strict NAT types reported on my devices. I'm wondering if this could possibly be due to some setup on the router, rather than a problem with the Three network?

**Comment by jamesmacwhite** [comment]
In theory if you are routed a public IP, this router shouldn't prevent direct access. There is a firewall enabled which may be causing issues.

**Comment by mattswatson** [comment]
Yeah, I'm not too sure what's going on to be honest. I definitely have a public IP, but the problem persists. I can't see anything in the firewall that should cause any issues, but I could be missing something.

I was wondering if it's possibly an issue with a double-NAT - perhaps the internal modem has a NAT, as well as the router itself?
